Added NCP1030 and NCP1031 PWM controllers

cafee opened the pull request:
Hello,

I've added NCP1030/NCP1031 PWM controllers symbols to switching regulators library.

Datasheet: https://www.onsemi.com/pub/Collateral/NCP1030-D.PDF

Screenshot of NCP1031DR - others symbols are similliar
![ncp1030](https://user-images.githubusercontent.com/8760751/55666267-a32e7380-584c-11e9-9100-6f98d000d1cb.jpg)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/KiCad/kicad-symbols?pullRequest=1714) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/KiCad/kicad-symbols?pullRequest=1714) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/KiCad/kicad-symbols?pullRequest=1714) it.</sub>

myfreescalewebpage commented:
@cafee thanks for this contribution and welcome ! Before we can review, can you sign the CLA please ? That's a mandatory condition to get your contribution accepted. Thanks, Joel

evanshultz commented:
Welcome and thanks for making a contribution to the official KiCad library!

Here are my review notes:
1. The description doesn't differentiate the parts. At the end of the existing description, add `, 1A` or `, 2A` for the switch current depending on the part number.
1. Then after that, add the package like `, SOIC-8` or `, DFN-8`.
1. Change CT and COMP to Passive type. Let's also do this for VDRAIN also since no other pin type will really help to catch issues as it's not a logic pin.
1. The DR symbol has the wrong footprint assigned. Should be 3.9x4.9mm package.
1. On the MN symbol, pin 9 must be visible and of Passive type. I don't see that it has an internal connection in the datasheet so it should be a visible pin named EP. Correct me if I'm wrong. (I also notice that it optionally can have pullback leads which is terribly annoying.)

If you have questions or need help, please review http://kicad-pcb.org/libraries/klc/ first and then ask here.

cafee commented:
Thanks for feedback.
I had addressed all notes from you and prepared next commit. 
To description i had added the targeted power range (3W, 6W respectively) and footprint.

evanshultz commented:
Thanks!

1. Power as the differentiator is good with me. Remove ` Up to` and ` package` from the description, and also add a hyphen before the pin count for the MN version.
2. The correct DR footprint is 'SOIC' and not just 'SO'. I'll look into why these footprints are in the library.
3. The MN version probably most commonly connects the EP to ground, so please that pin at the bottom and please post a new screenshot for easy review.

myfreescalewebpage commented:
@cafee are you still working on this subject ?
